src/ipt_main.rb

```diff
@@ -40,6 +40,24 @@ def valid_protocol?(protocol)
   protocol =~ /^(tcp|udp)$/i ? true : false
 end
 
+def release_ip_and_check(private_ip, public_ip = nil)
+  released_ip = $nat.release_ip private_ip, public_ip
```

Comment:
That's true. I would like to changes this in branch [`task_create_gem`](https://github.com/dice-cyfronet/ip-wrangler/tree/task_create_gem). See PR #4